arch/x86/Kconfig

```diff
@@ -26,6 +26,7 @@ config CPU_ATOM
 	bool
 	select CMOV
 	select CPU_HAS_FPU
+	select FP_DOUBLE_PRECISION
```

Comment:
I suppose, this mean that x86 has double precision always.
However: is this option actually used for x86?

I updated the comment of CPU_HAS_FPU reflecting my question, here.